examples/feature-move-animation.js

```diff
@@ -0,0 +1,205 @@
+  'uEeFymAssAkdAmhAyTcVkFeEoKiH}l@kp@wg@sj@ku@ey@uh@kj@}EsFmG}Jk^_r@_f@m',
+  '~@ym@yjA??a@cFd@kBrCgDbAUnAcBhAyAdk@et@??kF}D??OL'
+].join('');
+
+var route = /** @type {ol.geom.LineString} */ (new ol.format.Polyline({
+  factor: 1e6
+}).readGeometry(polyline, {
+  dataProjection: 'EPSG:4326',
+  featureProjection: 'EPSG:3857'
+}));
+
+var routeCoords = route.getCoordinates();
+var routeLength = routeCoords.length;
+
+var routeFeature = new ol.Feature({
+  type: 'route',
+  geometry: route
+});
+var geoMarker = new ol.Feature({
+  type: 'geoMarker',
+  geometry: new ol.geom.Point(routeCoords[0])
+});
+var startMarker = new ol.Feature({
+  type: 'icon',
+  geometry: new ol.geom.Point(routeCoords[0])
+});
+var endMarker = new ol.Feature({
+  type: 'icon',
+  geometry: new ol.geom.Point(routeCoords[routeLength - 1])
+});
+
+var styles = {
+  'route': new ol.style.Style({
+    stroke: new ol.style.Stroke({
+      width: 6, color: [237, 212, 0, 0.8]
+    })
+  }),
+  'icon': new ol.style.Style({
+    image: new ol.style.Icon({
+      anchor: [0.5, 1],
+      src: 'data/icon.png'
+    })
+  }),
+  'geoMarker': new ol.style.Style({
+    image: new ol.style.Circle({
+      radius: 7,
+      snapToPixel: false,
+      fill: new ol.style.Fill({color: 'black'}),
+      stroke: new ol.style.Stroke({
+        color: 'white', width: 2
+      })
+    })
+  })
+};
+
+var vectorLayer = new ol.layer.Vector({
+  source: new ol.source.Vector({
+    features: [routeFeature, geoMarker, startMarker, endMarker]
+  }),
+  style: function(feature, resolution) {
+    // hide geoMarker if animation is active
+    if (animating && feature.get('type') === 'geoMarker') {
+      return [];
+    }
+    return [styles[feature.get('type')]];
+  }
+});
+
+var center = [-5639523.95, -3501274.52];
+var map = new ol.Map({
+  target: document.getElementById('map'),
+  loadTilesWhileAnimating: true,
+  view: new ol.View({
+    center: center,
+    zoom: 10,
+    minZoom: 2,
+    maxZoom: 19
+  }),
+  layers: [
+    new ol.layer.Tile({
+      source: new ol.source.BingMaps({
+        imagerySet: 'AerialWithLabels',
+        key: 'Ak-dzM4wZjSqTlzveKz5u0d4IQ4bRzVI309GxmkgSVr1ewS6iPSrOvOKhA-CJlm3'
+      })
+    }),
+    vectorLayer
+  ]
+});
+
+var moveFeature = function(event) {
+  var vectorContext = event.vectorContext;
+  var frameState = event.frameState;
+
+  if (animating) {
+    var elapsedTime = frameState.time - now;
+    // here the trick to increase speed is to jump some indexes
+    // on lineString coordinates
+    var index = Math.round(speed * elapsedTime / 1000);
+
+    if (index >= routeLength) {
+      stopAnimation(true);
+      return;
+    }
+
+    var currentPoint = new ol.geom.Point(routeCoords[index]);
+    var feature = new ol.Feature(currentPoint);
+    vectorContext.drawFeature(feature, styles.geoMarker);
+  }
+  // tell OL3 to continue the postcompose animation
+  frameState.animate = true;
```

Comment:
I think we should encourage folks to use [`map.render()`](http://openlayers.org/en/master/apidoc/ol.Map.html#render) here instead.